src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -198,22 +199,9 @@ private async Task<Guid> StartApplication(Guid signinId)
 
                 if (applicationId != Guid.Empty)
                 {
-                   await SavePreambleInformation(applicationId, applicationDetails);
-                   _logger.LogDebug("Preamble information saved");
-
-                    if (applicationDetails.UkrlpLookupDetails.VerifiedByCompaniesHouse)
-                    {
-                        await SaveCompaniesHouseInformation(applicationId, applicationDetails);
-                        _logger.LogDebug("Companies House information saved");
-                    }
-
-                    if (applicationDetails.UkrlpLookupDetails.VerifiedbyCharityCommission)
-                    {
-                        await SaveCharityCommissionInformation(applicationId, applicationDetails);
-                        _logger.LogDebug("Save Charity Commission information saved");
-                    }
```

Comment:
As this conditional logic is now removed; does that mean the new code can handle it correctly still?

Reply:
never mind I see it's done via StorePreambleInformation 😀